.gitignore

```diff
@@ -6,3 +6,4 @@ build/reports
 dist/
 spec/*.js
 spec/*.map
+.vscode
```

Comment:
This shouldn't be ignored; these files are needed for anyone working in vscode and want to debug (breakpoints) by pressing f5